doc/zhCN/user/source/styling/css/tutorial.rst

```diff
@@ -0,0 +1,511 @@
+            </Rule>
+            <Rule>
+              <Title>2M - 4M</Title>
+              <ogc:Filter>
+                <ogc:PropertyIsBetween>
+                  <ogc:PropertyName>PERSONS</ogc:PropertyName>
+                  <ogc:LowerBoundary>
+                    <ogc:Literal>2000000</ogc:Literal>
+                  </ogc:LowerBoundary>
+                  <ogc:UpperBoundary>
+                    <ogc:Literal>4000000</ogc:Literal>
+                  </ogc:UpperBoundary>
+                </ogc:PropertyIsBetween>
+              </ogc:Filter>
+              <PolygonSymbolizer>
+                 <Fill>
+                    <!-- CssParameters allowed are fill (the color) and fill-opacity -->
+                    <CssParameter name="fill">#FF4D4D</CssParameter>
+                    <CssParameter name="fill-opacity">0.7</CssParameter>
+                 </Fill>     
+              </PolygonSymbolizer>
+            </Rule>
+            <Rule>
+              <Title>&gt; 4M</Title>
+              <!-- like a linesymbolizer but with a fill too -->
+              <ogc:Filter>
+                <ogc:PropertyIsGreaterThan>
+                 <ogc:PropertyName>PERSONS</ogc:PropertyName>
+                 <ogc:Literal>4000000</ogc:Literal>
+                </ogc:PropertyIsGreaterThan>
+              </ogc:Filter>
+              <PolygonSymbolizer>
+                 <Fill>
+                    <!-- CssParameters allowed are fill (the color) and fill-opacity -->
+                    <CssParameter name="fill">#4D4DFF</CssParameter>
+                    <CssParameter name="fill-opacity">0.7</CssParameter>
+                 </Fill>     
+              </PolygonSymbolizer>
+            </Rule>
+            <Rule>
+              <Title>Boundary</Title>
+              <LineSymbolizer>
+                <Stroke>
+                  <CssParameter name="stroke-width">0.2</CssParameter>
+                </Stroke>
+              </LineSymbolizer>
+              <TextSymbolizer>
+                <Label>
+                  <ogc:PropertyName>STATE_ABBR</ogc:PropertyName>
+                </Label>
+                <Font>
+                  <CssParameter name="font-family">Times New Roman</CssParameter>
+                  <CssParameter name="font-style">Normal</CssParameter>
+                  <CssParameter name="font-size">14</CssParameter>
+                </Font>
+                <LabelPlacement>
+                  <PointPlacement>
+                    <AnchorPoint>
+                      <AnchorPointX>0.5</AnchorPointX>
+                      <AnchorPointY>0.5</AnchorPointY>
+                    </AnchorPoint>
+                  </PointPlacement>
+                </LabelPlacement>
+              </TextSymbolizer>
+            </Rule>
+         </FeatureTypeStyle>
+        </UserStyle>
+        </NamedLayer>
+    </StyledLayerDescriptor>
+
+现在，让我们从完成相同任务的 CSS 文件开始。
+
+首先，进入样式页面，点击 :guilabel:`add a new style` 链接来开始新样式。在“新样式”页面中，执行以下操作::
+
+* 将新样式命名为任何您喜欢的名称，例如 ``csstutorial``
```

Comment:
One of the screenshots says "csstorial" though.

Reply:
The screenshot has been updated.